README.md

```diff
@@ -37,16 +37,166 @@ NOTE: we suggest using virtualenv to sandbox your setup.
 - [VALIDATOR_GUIDE.md](https://github.com/ethereum/casper/blob/master/VALIDATOR_GUIDE.md):
 information about implementing a Casper FFG validator.
 
-## Contribute
+## Tests
 
-Feel free to ask questions in our [Gitter room](https://gitter.im/ethereum/casper) or open an [issue](https://github.com/ethereum/casper/issues) for feature requests or bug reports. Feel free to make a PR!
+```bash
+pytest tests
+```
 
-## License
+## Version of Vyper to Compile Casper
+We recommend using the vyper version installed as a dependency from requirements.txt. 
+NOTE: The latest version of Vyper throws an error (line 16) when compiling the Casper contract due to a syntax change of "units".
+NOTE: this not needed if steps above followed
+```bash
+git clone https://github.com/ethereum/vyper.git@248c723288e84899908048efff4c3e0b12f0b3dc
+```
+The guide on how to install Vyper can be found [here](https://github.com/ethereum/vyper#installation).
 
-[UNLICENSE](LICENSE)
 
-## Tests
+## How to Build and Deploy Casper Contract
+
+In this example we will be using [eth-tester](https://github.com/ethereum/eth-tester).
+
 
+### Step 1: Imports
 ```bash
-pytest tests
+import os
+import rlp
+
+from decimal import (
+    Decimal,
+)
+
+import eth_tester
+
+from eth_tester import (
+    EthereumTester,
+    PyEVMBackend
+)
+from web3.providers.eth_tester import (
+    EthereumTesterProvider,
+)
+
+from web3 import (
+    Web3,
+)
+
+from vyper import (
+    compiler,
+    utils as vyper_utils,
+)
+
+```
+
+### Step 2: Raise the Gas Limit
+
+```bash
```

Comment:
Just note that the contract costs a lot of gas (can give a number) and that if they are deploying via a normal CREATE tx, then they need to make sure the block gas limit is high enough